packages/flutter_tools/test/general.shard/build_system/targets/web_test.dart

```diff
@@ -44,10 +44,10 @@ void main() {
       PackageMap.globalPackagesPath = packagesFile.path;
       globals.fs.currentDirectory.childDirectory('bar').createSync();
 
-      environment = Environment(
+      environment = Environment.test(
+         globals.fs.currentDirectory,
```

Comment:
Is this whitespace wrong?  I can't tell.

Reply:
Off by 1

Reply:
Fixed